src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -228,12 +217,12 @@ end
 function rhs!(du_ode, u_ode, semi::SemidiscretizationHyperbolicParabolic, t)
   @unpack mesh, equations, initial_condition, boundary_conditions, source_terms, solver, cache = semi
 
-  u  = wrap_array(u_ode,  mesh, equations, solver, cache)
-  du = wrap_array(du_ode, mesh, equations, solver, cache)
+  u  = wrap_array(u_ode,  mesh, equations.equations_hyperbolic, solver, cache)
+  du = wrap_array(du_ode, mesh, equations.equations_hyperbolic, solver, cache)
```

Comment:
I think you could avoid this if you implemented `nvariables(equations)` for the combined system

Reply:
I believe we did implement `nvariables`. Let me try removing this